### PR TITLE
Stop external medications from being ingested 

### LINKED ISSUE #29612: Applying external medications acts as though the medication were eaten.
**Describe the bug**
Applying external medications acts as though the medication were eaten.
E.g. applying bandages supress hunger as though they were a meal.

**To Reproduce**  
1. Get hungry
2. Get damaged
3. Apply bandages 

Hunger will be supressed.

**Expected behavior**  
Hunger not supressed.

**Proposal**
player::consume_med() should refrain from calling consume_effects() for non-ingested medications, of which there are many. This requires a new comestible flag.

### PR DESCRIPTION
**Pull request by Yunatan**
#### Summary
```SUMMARY: Bugfixes "Applying external medications no longer acts as though the medication were eaten."```

#### Purpose of change
Fixes #29612 - Applying external medications acts as though the medication were eaten.

#### Describe the solution
I have added a new comestible flag "NO_INGEST" to all parenteral medications to refrain ``player::consume_med()`` from calling ``consume_effects()`` for non-ingested medications.

#### Describe alternatives you've considered
Implement full-blown routes of administration differentiation. 

#### Additional context
I have also changed pink tablet to pink tab to emphasize that it is not swallowed 🖃

### CODE REVIEW

**Comment by Yunatan** [comment]
This will not make non-edible comestibles stop populating the eat menu, that would be a separate issue.

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -46,7 +46,8 @@
     "material": "plastic",
     "symbol": "!",
     "color": "pink",
-    "use_action": "ADRENALINE_INJECTOR"
+    "use_action": "ADRENALINE_INJECTOR",
+    "flags": "NO_INGEST"
```
Comment (anothersimulacrum) [inline data/json/items/comestibles/med.json]:
```suggestion
    "flags": [ "NO_INGEST" ]
```

File: data/json/items/comestibles/mutagen.json [hunk 2]
```diff
@@ -38,7 +38,8 @@
     "stim": -5,
     "addiction_potential": 6,
     "container": "test_tube",
-    "tools_needed": "syringe"
+    "tools_needed": "syringe",
+    "flags": "NO_INGEST"
```
Comment (anothersimulacrum) [inline data/json/items/comestibles/mutagen.json]:
```suggestion
    "flags": [ "NO_INGEST" ]
```

**Comment by ktoma36** [comment]
Does this allow for mycus mutated characters to use items such as bandages or is that still a no go?

**Comment by adamkad1** [comment]
mycus needs no bandages

**Pull request merged**